### ISSUE TITLE
[App Crash] My Resonite Keeps Crashing

### ISSUE THREAD

**Issue by AkagiVoid** [Yellow-Dog-Man/Resonite-Issues#2480]
### Describe the bug?

Every time I load into the world or other worlds, after 3 to 5 minutes from when Resonite is launched, it would crash. *confused* @w@
Although if I stay in my Home World it does not crash as often, so I'm confused. @w@

### To Reproduce

Launches Resonite from STEAM, waits for world to load, starts moving around, finds a place to start tinkering, and the game crashes as I'm looking through my Items in the UI for no reason at all. >w>
(OR)
Launches Resonite from STEAM, waits for world to load, brings up the UI, launches another blank grid world to start building in, gets there, grabs dev tool, and starts looking through the menu options, and game crashes for no reason at all. >w>

### Expected behavior

App does not suddenly crash when doing everyday normal operations. >w>

### Screenshots

![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/48104310/06166e36-a996-49e6-8f4b-c952fe77d05d)
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/48104310/4dab6a0d-5ff3-416c-997d-51fa0649b538)

Video zip file as well in webm and mp4, [Mediafire](https://www.mediafire.com/file/3zztc9vle5padbb/ResoniteBeta2024741347_AppCrashVideos.zip/file). >w<

### Resonite Version Number

Beta 2024.7.4.1347

### What Platforms does this occur on?

Windows

### What headset if any do you use?

N/A

### Log Files

Steam App Folder Logs:
[steamappcommonResonite_Logs.zip](https://github.com/user-attachments/files/16112486/steamappcommonResonite_Logs.zip)

Player Logs:
[AppDataLocalLowYellowDogManStudios_PlayerLogs.zip](https://github.com/user-attachments/files/16112500/AppDataLocalLowYellowDogManStudios_PlayerLogs.zip)

Crash dumps:
[AppDataLocalTempYellowDogManStudiosResonite_Crashes.zip](https://github.com/user-attachments/files/16112733/AppDataLocalTempYellowDogManStudiosResonite_Crashes.zip)

### Additional Context

My Resonite is completely vanilla btw. >w<

Desktop specs:
OS: Windows 10 Pro 64x,
Launcher App: STEAM,
MOBO: Gigabyte X470 AORUS GAMING 7 WIFI (rev. 1.0),
CPU: Ryzen 7 2700x,
GPU: MSI GTX 1080 TI Gaming X Trio,
Ram: DDR4 16GB,
Storage: SSDs 256GBx2 Silicon Power, HDD 2TB HP. >w>

**Comment by Frooxius** [comment]
Can you include crash dumps please? They will be at `C:/Users/shizu/AppData/Local/Temp/Yellow Dog Man Studios/Resonite/Crashes`

**Comment by AkagiVoid** [comment]
> Can you include crash dumps please? They will be at `C:/Users/shizu/AppData/Local/Temp/Yellow Dog Man Studios/Resonite/Crashes`

TY for the fast response, I've updated the main post with the crash dumps. >w<

**Comment by AkagiVoid** [comment]
I seemed to have resolved the issue my self. :3
Steps I did:
Clear the Temp folder in AppData > Local, use windows built in "Disk Cleaner", Use 3rd party app "CCleaner64" to fix registry errors, then did a fresh re-install the GPU drivers, and the issue magically resolved its self. X3

Will re-open dis ticket or make a new ticket if the problem pops up again and the methods/steps above do not resolve it. X3
